packages/thumbprint-react/components/Avatar/subcomponents/badge.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styles from './badge.module.scss';
+
+interface ConfigItem {
+    top?: number;
+    right?: number;
+    size: number;
+}
+
+type BadgeType = 'userOnline' | 'userChecked' | 'entityOnline';
+
+// TODO(giles): remove all "checked" configs once we delete the isChecked prop
+const sizeClasses: {
```

Comment:
Would this be more legible is the type were above the `const`? Took me a sec to figure out what was going on. (Feel free to leave as is—just want to get your opinion.)

Reply:
Good idea, i moved it out